db/kv_dense_channel_storage.go

```diff
@@ -378,7 +379,7 @@ func (l *DenseBlockList) populateForRange(partitionRange base.PartitionRange) er
 	for partitionRange.SinceBefore(l.ValidFrom()) {
 		err := l.LoadPrevious()
 		if err != nil {
-			return fmt.Errorf("Unable to load previous block list: %v", err)
+			return pkgerrors.Wrapf(err, "Unable to load previous block list")
```

Comment:
Candidate for wrapping in LoadPrevious()?

Reply:
I pushed this as far down as I could until I hit the `Bucket` interface.  I didn't try to go further, since to ensure it was wrapped (and got a stack trace), it would have be changed in walrus too.

If it were "double wrapped" in `bucket_gocb.go`, I don't think it would be a problem, and I think it might even give us a longer stacktrace in that case.